core/src/main/scala/za/co/absa/fadb/statushandling/UserDefinedStatusHandling.scala

```diff
@@ -14,23 +14,21 @@
  * limitations under the License.
  */
 
-package za.co.absa.fadb.slick
+package za.co.absa.fadb.statushandling
```

Comment:
I was thinking more like `statushandling` -> `status_handling` for naming modules generally speaking, so with `_` as 'delimiter', but I guess I don't care that much :D

Reply:
That's actually exactly against naming conventions (shouldn't have `_`) and the existence of #34 (`naming_conventions` package needs to go)